config/config.go

```diff
@@ -62,6 +62,7 @@ type Configuration struct {
 	SmappVersion             string `mapstructure:"smapp-version"`
 	TLSKey                   string `mapstructure:"tls-key"`
 	TLSCert                  string `mapstructure:"tls-cert"`
+	KeepLogsMetrics          bool   `mapstructure:"keep-logs-metrics"`
```

Comment:
How do we suppose to delete logs/metrics in the aftermath? If manually, then a separate command solely for that should probably be added.

Reply:
You can then use the deleteNetwork sub command without --keep-logs-metrics CLI option. It will delete everything.